docs/index.bs

```diff
@@ -3259,10 +3266,17 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
       :: {{RouterSource}} or null
 
       1. [=list/For each=] |rule| of |serviceWorker|'s [=service worker/list of router rules=]:
-          1. If |rule|["{{RouterRule/condition}}"]["{{RouterCondition/urlPattern}}"] does not [=map/exist=], continue.
-          1. Let |rawPattern| be |rule|["{{RouterRule/condition}}"]["{{RouterCondition/urlPattern}}"].
-          1. Let |pattern| be the result of running <a>Parse URL Pattern</a> algorithm passing |rawPattern| and |serviceWorker|.
-          1. If running [=match=] with |pattern| and |request|'s [=request/URL=] returns null, [=continue=].
+
+          Note: if there are multiple conditions in a rule, all conditions should be matched to return the source.
```

Comment:
```suggestion
          Note: if there are multiple conditions in a rule, all conditions will be matched to return the source.
```

Try to avoid keywords like "must" or "should" inside non-normative explanatory notes.

Reply:
done.